README.md

```diff
@@ -51,27 +51,44 @@ Once the implementation of a piece of functionality is considered to be bug
 free and properly documented (both API docs and an example script),
 it can be incorporated into the master branch.
 
-To run the tests using `pytest`, you need to have the cloned mne-python repository
-with a local pip-install instead of the mne-python package from pypi. Update your
-installation as follows:
+To help developing mne-bids, you will need a few adjustments to your
```

Comment:
you're missing ` around `mne-bids`

Reply:
you mean `'` or `?

Reply:
The latter but I couldn't find a way to type it :-P